### ISSUE TITLE
ip range

### ISSUE THREAD

**Issue by davidhold** [angryip/ipscan#128]
why does it always default to 192.168.1.1 range

why not let it remember last used or set it in preferences

i us apple airport extreme

numbering starts at 10.0.1.0 
![range](https://user-images.githubusercontent.com/6422071/35782591-be825a48-09bf-11e8-8812-f145ecd5e2d2.jpg)

**Comment by angryziber** [comment]
Actually, it detects the range of your local network (and takes the first one if there are several network interfaces).

Press the "IP^" button just after start to choose the interface. I guess you have 192.168.1.x configured somewhere.

**Comment by maxily** [comment]
@angryziber so the range is based on ip address coming from a detected network interface? Does that mean that started on the range search goes via the "selected" interface only? and doesn't spam to other interfaces?

Wonder if it would be useful to someone else except me to show the selected network interface explicitly on UI? Eg. instead of Hostname field

**Comment by angryziber** [comment]
Yes, ipscan defaults to the first network interface that you have enabled
in your machine.

Have you tried clicking the `IP^` button next to your hostname? It should
list all your interfaces, so you can select the one you want at this time.

There are also favorites.

Maybe remembering the last one used is also a good idea, but could be wrong
for people hopping between different networks.

**Comment by maxily** [comment]
Yep, I selected the network interface I wanted to use. Thought it's slightly confusing that the selected interface name is not shown except in the dropdown `IP`
